test/MyFirstApiConsume.test.js

```diff
@@ -69,30 +24,46 @@ describe('First Api Tests', () => {
 	});
 
 	it('Consume HEAD Service', async () => {
-		const response = await axios.get('https://httpbin.org/headers');
+		const response = await axios.head('https://httpbin.org/headers');
 
 		expect(response.status).to.equal(StatusCodes.OK);
-		expect(response.data).to.have.property('headers');
+		expect(response.config.url).to.have.string('/head');
+		expect(response.data).to.be.empty;
 	});
 
 	it('Consume PATCH Service', async () => {
-		const response = await axios.patch('https://httpbin.org/patch');
+		const query = {
+			name: 'Diego Alejandro',
+			surname: 'Cortes Roa',
+			career: 'Multimedia Engineer',
+		};
+
+		const response = await axios.patch('https://httpbin.org/patch', query);
 
 		expect(response.status).to.equal(StatusCodes.OK);
 		expect(response.config.url).to.have.string('/patch');
+		expect(JSON.parse(response.data.data)).to.eql(query);
 	});
 
 	it('Consume PUT Service', async () => {
-		const response = await axios.put('https://httpbin.org/put');
+		const query = {
+			name: 'Diego Alejandro',
+			surname: 'Cortes Roa',
+			career: 'Multimedia Engineer',
+		};
+
+		const response = await axios.put('https://httpbin.org/put', query);
 
 		expect(response.status).to.equal(StatusCodes.OK);
 		expect(response.config.url).to.have.string('/put');
+		expect(JSON.parse(response.data.data)).to.eql(response.data.json);
```

Comment:
You are verifying two fields of the response, validate the data or json property with the data you send in axios